### PR TITLE
Use merged bundle config to allow overrides

### PR DESCRIPTION
**Pull request by TamasSzigeti**
Hi, awesome package!

I found this small bug when I tried to add extra namespaces on test env for some fixture queries/types

### CODE REVIEW

**Review by devmaslov — APPROVED** [review]
@TamasSzigeti, thanks for bringing this up. Looks good to me!

@moufmouf I can help maintaining this repo as well. 
Could you share more info on our release process? 
At least the following questions interest me:
- Who can do this?
- How often are we doing releases? After each PR merge to master?

**Comment by TamasSzigeti** [comment]
@moufmouf It is really great that you involved @devmaslov and @oojacoboo as maintainers, but maybe they also need to be empowered to merge PRs and cut releases to make things move a bit more swiftly? Or is there anything blocking for example this PR to be merged and released?

**Comment by moufmouf** [comment]
Oh jee, sorry, I completely missed that PR.

@devmaslov you are maintainer of this package.
I completely trust you to merge and tag a new release if you deem it necessary.

One important thing about graphqlite-bundle: I think it is better if we keep the version number in sync with GraphQLite.

So graphqlite-bundle 4.1.x is compatible with graphqlite 4.1.x.
This means that graphqlite-bundle does not strictly follow semver. For instance, we could add a new feature in a "patch" release.

**Pull request merged**
